addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,647 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="bridge">
+		<label>RFXCOM Transceiver</label>
+		<description>This is universal RFXCOM transceiver bridge for manual configuration purposes.</description>
+
+		<config-description>
+			<parameter name="serialPort" type="text">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>Serial port where RFXCOM transceiver is connected.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="ignoreConfig" type="boolean">
+				<context></context>
+				<label>Ignore RFXCOM transceiver configuration</label>
+				<description>Fully ignore RFXCOM transceiver configuration. Binding assume that RFXCOM transceiver is preconfigured e.g. via RFXcom Manager.</description>
+				<required>false</required>
+				<default>true</default>
+			</parameter>
+			<parameter name="setMode" type="text">
+				<context></context>
+				<label></label>
+				<description>RFXCOM transceiver set mode command. Command should be in hexadecimal string format and 28 characters (14 bytes) long. If set mode command is given, individual messages configurations are ignored.</description>
+				<required>false</required>
+			</parameter>
+			<parameter name="transceiverType" type="text">
+				<context></context>
+				<label></label>
+				<description>RFXCOM transceiver type.</description>
+				<required>false</required>
+				<default>433.92MHz</default>
+				<options>
+					<option value="310MHz">310MHz</option>
+					<option value="315MHz">315MHz</option>
+					<option value="433.92MHz receiver only">433.92MHz receiver only</option>
+					<option value="433.92MHz">433.92MHz</option>
+				</options>
+			</parameter>
+			<parameter name="enableUndecoded" type="boolean">
+				<context></context>
+				<label>Enable undecoded messages</label>
```

Comment:
Please keep the label texts as short as possible, e.g. remove the "Enable" word (throughout this file)